src/components/project/hero.tsx

```diff
@@ -1,5 +1,259 @@
 import Image from "next/image";
 import { IProjectData, Project } from "../../interfaces/project";
```

Comment:
Remove unused import 
```suggestion
import { Project } from "../../interfaces/project";
```